x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_add_bulk_to_timeline.tsx

```diff
@@ -62,15 +66,28 @@ export const useAddBulkToTimelineAction = ({
   scopeId,
 }: UseAddBulkToTimelineActionProps) => {
   const [disableActionOnSelectAll, setDisabledActionOnSelectAll] = useState(false);
+  const { newDataViewPickerEnabled } = useEnableExperimental();
+
+  const { dataView: experimentalDataView } = useDataView(scopeId);
+  const experimentalBrowserFields = useBrowserFields(scopeId);
+  const experimentalSelectedPatterns = useSelectedPatterns(scopeId);
 
-  const {
+  let {
     browserFields,
     dataViewId,
     sourcererDataView,
     // important to get selectedPatterns from useSourcererDataView
     // in order to include the exclude filters in the search that are not stored in the timeline
     selectedPatterns,
   } = useSourcererDataView(scopeId);
+
+  if (newDataViewPickerEnabled) {
+    dataViewId = experimentalDataView.id ?? '';
```

Comment:
can we default to an empty string if all the other parameters are provided in the reducer and not do these nullish checks everywhere the id is called?

Reply:
eventually we will only pass the spec reference around, I imagine.  Unfortunately this is how the DataViewSpec looks like 😭